### PR TITLE
fix: improve instruction for adding uno.css

### PR DESCRIPTION
**Pull request by AnnaRybkina**
We are not advising correctly about adding `uno.css` to the main entry. Hope this change can add some clarification

Background: https://sch-chat.slack.com/archives/C04EK4XHUJJ/p1687447545757319?thread_ts=1687442803.982849&cid=C04EK4XHUJJ

### CODE REVIEW

**Review by BalbinaK — APPROVED** [review]
🤩

File: docs/getting-started/developers/index.md [hunk 1]
```diff
@@ -118,6 +118,10 @@ For more examples how to configure other building tools, please refer to the [ex
 
 #### Add uno.css to your main entry
 
+Depending on the build tool you have chosen this step can look different. The example below is applicable if you build unocss together with vite. If
```
Comment (BalbinaK) [inline docs/getting-started/developers/index.md]: It's a very tiny thing but I was wondering, what do you think if we always called the tool `UnoCSS` within Warp docs? I'm mostly thinking about consistency. The same with `Vite` (capital V) and `PostCSS`. Maybe linking to [PostCSS](https://www.npmjs.com/package/postcss) site might also make sense? Wdyt?

Comment (AnnaRybkina) [reply docs/getting-started/developers/index.md]: I agree! I fix.
I don't know about linking to PostCSS. A lot of our consumers are already using it and we are not recommending the tool, just providing an example 🤔 We do not link to Vite either

File: docs/getting-started/developers/index.md [hunk 2]
```diff
@@ -118,6 +118,10 @@ For more examples how to configure other building tools, please refer to the [ex
 
 #### Add uno.css to your main entry
 
+Depending on the build tool you have chosen this step can look different. The example below is applicable if you build unocss together with Vite. If
```
Comment (BalbinaK) [inline docs/getting-started/developers/index.md]: UnoCSS? 😇

Comment (AnnaRybkina) [reply docs/getting-started/developers/index.md]: ah, took me a while to find it 👁️ I fix, tnx

**Pull request merged**